python_modules/libraries/dagster-aws/dagster_aws_tests/aws_credential_test_utils.py

```diff
@@ -26,6 +26,11 @@ def get_aws_creds():
         return creds
 
     except Exception as e:
+        if os.getenv("BUILDKITE"):
+            import pytest
+
+            pytest.skip("Integration test")
```

Comment:
* I think we should try to have a more informative message - `"[integration test] could not load aws creds, skipping"`
*  I have some concern around this skip triggering when we don't want it to and no one noticing, can we add another bit of gating like an env var `SKIP_ON_INTEGRATION_FAIL` or something ?